api/MediaDevices.json

```diff
@@ -208,40 +211,65 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/getUserMedia",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": "53",
+              "notes": [
+                "Older versions of Chrome and Opera implement navigator.webkitGetUserMedia, the prefixed version of the legacy navigator.getUserMedia method. From version 47 to 52, the promise-based interface was only available through the adapter.js polyfill, or using the flag chrome://flags/#enable-experimental-web-platform-features. Starting with version 53, the promise-based interface is on by default, though that interface is still not available through navigator.",
```

Comment:
Looking at the other data, I think most of this doesn't apply to WebView (unlike the Chromes and Operas). I'd drop the notes here entirely

Reply:
I'm not sure about this as I haven't tested this. But looking at the superscripted text(53[1][3]) and the related legend for the *Android Webview* field - [see this](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#Browser_compatibility), it looks like this could be related. Again I'm not sure.

Reply:
I'm not sure about this as I haven't tested this. But looking at the superscripted text(53[1][3]) and the related legend for the *Android Webview* field - [see this](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#Browser_compatibility), it looks like this could be related. Again I'm not sure.

Reply:
I've removed the notes.

Reply:
Ah, just to clear this up: the reason I mentioned removing the note is that the older API doesn't need be documented here, the preference doesn't apply to WebView (there's no preferences for WebView, unlike Chrome and Opera on Android), and the last bit was covered by the "53" value anyway.